include/swift/Driver/Job.h

```diff
@@ -115,7 +115,7 @@ class Job {
   EnvironmentVector ExtraEnvironment;
 
   /// Whether the job wants a list of input or output files created.
-  FilelistInfo FilelistFileInfo;
+  std::vector<FilelistInfo> FilelistFileInfos;
```

Comment:
I'm a little unhappy about this because I suspect it'll all collapse back down to just one file list when we're done, but I guess it does make sense for now.

Reply:
Even with the OFM I think it winds up being ... 3? inputs (for order), primary-inputs (for subset-selection), and outputs (though maybe only in wmo?).

Reply:
@jrose-apple or .. are you thinking that maybe the multiple-FilelistFileInfos scenario only occurs in the BatchModeJob subclass, so we should move it there?

Reply:
Oh, yeah, I guess. Okay, this is fine.